openhabian-setup.sh

```diff
@@ -48,10 +48,11 @@ elif [[ "$1" = "unattended_debug" ]]; then
 else
   INTERACTIVE=1
 fi
+export UNATTENDED SILENT INTERACTIVE SCRIPTNAME REPOSITORYURL CONFIGFILE
```

Comment:
Why do you export all these variables?

Reply:
shellcheck reminds me to although it also says it cannot fully test if they're used elsewhere.
May not be necessary but doesn't do harm.

Reply:
Make a shellcheck exception here. This will just confuse the reader.

Reply:
who is browsing that if not for this review purpose only , are you sleepless or is it the Readers of the Lost Source ? ;-)
Seriously, I can't tell you the location in the code where this would cause issues yet, but I didn't feel 100% sure we would never need to have this exported anywhere. Recently I almost re-implemented configuring one of the variables before realizing it's already there.
IMHO it's better to export by default

Reply:
As this is the last but one diff line, does that mean you're done reviewing ?
There many many changed lines above this you didn't comment on (then again it's often SC2181...)

Reply:
No, it happen to be the "last" file. I have still more then half of the file left but I am working on it. I will notice you when I am done. 👍

Reply:
The `export` command would not cause programmatic issues but will clutter the code and increase the burden of working with the codebase. 
This will be one of these lines people i future can't trace to a requirement and then be afraid to change/remove. All code should have a purpose therefore I want an `Shellcheck disable` here.